phones/tests/models_tests.py

```diff
@@ -251,52 +272,61 @@ def test_create_duplicate_relaynumber_raises_error(phone_user, mocked_twilio_cli
     mock_messages_create.assert_not_called()
 
 
+@pytest.fixture
+def real_phone_us(phone_user, mock_twilio_client):
+    """Create a US-based RealPhone for phone_user, with a reset twilio_client."""
+    real_phone = RealPhone.objects.create(
+        user=phone_user,
+        number="+12223334444",
+        verified=True,
+        verification_sent_date=datetime.now(timezone.utc),
+    )
+    mock_twilio_client.messages.create.assert_called_once()
+    mock_twilio_client.messages.create.reset_mock()
+    return real_phone
+
+
 def test_create_relaynumber_creates_twilio_incoming_number_and_sends_welcome(
-    phone_user, mocked_twilio_client
+    phone_user, real_phone_us, mock_twilio_client
 ):
+    """A successful relay phone creation sends a welcome message."""
     twilio_incoming_number_sid = "PNXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-    mock_twilio_client = mocked_twilio_client
     mock_messages_create = mock_twilio_client.messages.create
     mock_number_create = mock_twilio_client.incoming_phone_numbers.create
     mock_number_create.return_value = SimpleNamespace(sid=twilio_incoming_number_sid)
     mock_services = mock_twilio_client.messaging.v1.services
 
-    real_phone = "+12223334444"
-    RealPhone.objects.create(user=phone_user, verified=True, number=real_phone)
-    mock_messages_create.assert_called_once()
-    mock_messages_create.reset_mock()
-
     relay_number = "+19998887777"
     relay_number_obj = RelayNumber.objects.create(user=phone_user, number=relay_number)
 
-    mock_number_create.assert_called_once()
-    call_kwargs = mock_number_create.call_args.kwargs
-    assert call_kwargs["phone_number"] == relay_number
-    assert call_kwargs["sms_application_sid"] == settings.TWILIO_SMS_APPLICATION_SID
-    assert call_kwargs["voice_application_sid"] == settings.TWILIO_SMS_APPLICATION_SID
-
-    mock_services.assert_called_once()
-    call_args = mock_services.call_args
-    assert call_args[0][0] == settings.TWILIO_MESSAGING_SERVICE_SID
-    mock_messaging_number_create = (
-        mock_twilio_client.messaging.v1.services().phone_numbers.create
+    mock_number_create.assert_called_once_with(
+        phone_number=relay_number,
+        sms_application_sid=settings.TWILIO_SMS_APPLICATION_SID,
+        voice_application_sid=settings.TWILIO_SMS_APPLICATION_SID,
+    )
```

Comment:
praise: nice clean-up! When I first wrote a bunch of these assertions, for some reason the `assert_called_once_with` method didn't work for me, so I ended up pulling out the `call_args.kwargs` property and asserting on that. I was probably just doing something wrong.